gateway-service/src/test/java/org/zowe/apiml/gateway/controllers/GatewayHomepageControllerTest.java

```diff
@@ -285,4 +287,75 @@ void givenZOSMFProviderRunning_whenHomePageCalled_thenHomePageModelShouldContain
         assertThat(actualModelMap, IsMapContaining.hasEntry("authIconName", "success"));
         assertThat(actualModelMap, IsMapContaining.hasEntry("authStatusText", "The Authentication service is running"));
     }
+
+    @Nested
+    class whenLoadMetricsServiceLink {
+        @Test
+        void givenMetricsServiceEnabledAndRunning_thenLinkEnabled() {
```

Comment:
Could you use include also the `when` to the test name? So we will follow the `given_when_then` format

Reply:
I thought the squad had decided to use `@Nested` to avoid repetition with the same `when`. @balhar-jakub can you confirm?

Reply:
This follows the format of other newer test suites like `CachingControllerTest.java`

Reply:
Yeah it seems correct, sorry, I didn't notice the `@Nested` annotation and didn't know about the new naming convention.
It looks good then